### ISSUE TITLE
could you actually train a new model?

### ISSUE THREAD

**Issue by chevalfouk** [mitmul/chainer-pspnet#2]
Hi, I am currently trying to train the cityscapes dataset with your code, but the result is miserable: still 0.5263158 (=1/19) class accuracy after 120 epochs. Apparently, the loss of training data is converged correctly, so it seems like a perfect over fitting. Since I used the same settings as yours,  i am wondering how you managed to reproduce the results(maybe i need less learning rate?). thanks in advance!
